apps/frontend/src/libs/hooks/use-click-outside/use-click-outside.hook.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from "~/libs/hooks/hooks.js";
+
+const useClickOutside = <T extends HTMLElement>(
```

Comment:
let's rename it to useHandleClickOutside and don't forget about file name 
```suggestion
const useHandleClickOutside = <T extends HTMLElement>(
```